md/BonitaBPM_platform_setup.md

```diff
@@ -102,20 +109,49 @@ If you want to modify the configuration of an already initialized Bonita BPM Pla
 Note that the *Platform Setup tool* does not need the Bonita BPM Server to be running for the configuration to be updated. However, the server needs to be restarted for the changes to take effect.
 :::
 
-
 ::: warning
 Keep in mind that the folder **`platform_conf/initial`** is not used anymore once the platform has been initialized for the first time. To update your configuration, only the **`platform_conf/current`** folder is taken into account.
 :::
 
-## Troubleshooting
 
----
+<a id="run_bundle_configure" />
+
+## Run the Bonita BPM bundle configuration (Tomcat 7 / Wildfly 9)
 
-**Problem**: When I run Platform setup tool, I get the exception: `locked by an other process`
+::: info
+Remember that in the [Tomcat](tomcat-bundle.md) and [Wildfly](wildfly-bundle.md) bundles, this is done automatically when calling `bonita-start.sh / .bat`
+:::
 
-**Possible cause**: The Platform setup tool tries to access the same database (H2) as a running Bonita BPM Bundle or Bonita BPM Studio.
+/!\ TODO: document the TEMPLATES mechanism /!\
```

Comment:
So I guess you should either remove this line either put `WIP` label on this PR so no one will  merge it for now